debian/buster/foreman/rules

```diff
@@ -47,13 +47,9 @@ build:
 	dh $@ --with=systemd
 
 override_dh_systemd_start:
-	dh_systemd_start --no-start --restart-after-upgrade --name=foreman.service --package=foreman-service
-	dh_systemd_start --no-start --restart-after-upgrade --name=foreman.socket --package=foreman-service
-	dh_systemd_start --no-start --name=dynflow-sidekiq@orchestrator --package=foreman-dynflow-sidekiq
-	dh_systemd_start --no-start --name=dynflow-sidekiq@worker --package=foreman-dynflow-sidekiq
+	dh_systemd_start --no-start --restart-after-upgrade --package=foreman-service
+	dh_systemd_start --no-start --restart-after-upgrade --package=foreman-dynflow-sidekiq
```

Comment:
this is technically a noop, as the package doesn't contain any restartable services (the template isn't restartable), so we *could* drop it…